force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -102,16 +133,19 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer countVowels(String str) {
         // Initialize the result Integer
-
-        /*
-        for () {
+        Integer numOfVowels = 0;
+        String vowels = 'aeiou';
+        for (Integer i = 0; i < str.length(); i++) {
             // get individual characters from the string - hint use substring or split
+            String chars = str.substring(i, i + 1);
             // check if the character is a vowel
+            if (vowels.contains(chars)) {
+                numOfVowels ++;
+                System.debug(numOfVowels);
+            }
             // if it is a vowel, increment the result Integer
         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        return numOfVowels; // Replace null with the variable you used to store the result
```

Comment:
A few things with this one. Any time you are checking for membership (is the char a member of the set of vowells) use a set it is one of the most performant data structures there is for that use case. 

I would name that variable char and not chars. Chars suggests a string. And always best to delete debug statements before you commit.  You don't want them to make it to Prod. 

```
 public static Integer countVowels(String str) {
        Integer vowelCount = 0;
        Set<String> vowels = new Set<String>{
                'a', 'e', 'i', 'o', 'u'
        };
        List<String> characters = str.split('');
        for (String character : characters) {
            if (vowels.contains(character.toLowerCase())) {
                vowelCount++;
            }
        }
        return vowelCount;
    }
    ```